packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -299,6 +319,54 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
     _addPointer(event);
   }
 
+  /// When multiple pointers down and move, there may be the following situations
+  /// by different platforms:
```

Comment:
What do you mean by "different platforms"? Do you mean Android and iOS? The following bulletins are not separated by platforms. And this algorithm seems to only fit iOS.